nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml

```diff
@@ -135,6 +137,7 @@ If this is checked, then the binaries will be built and pushed to chacra even if
     scm:
       - nfs-ganesha
       - nfs-ganesha-debian
+      - ntirpc
```

Comment:
@alfredodeza this missing line was what was causing the ntirpc scm to not be acknowledged.